composables/projectProperties.ts

```diff
@@ -28,10 +32,21 @@ export function useProjectProperties() {
    */
   async function fetchProjectProperties(locale: string) {
     try {
-      return (
-        cachedProjectProperties.value[locale] ||
-        (await $fsxaApi.fetchProjectProperties({ locale }))
-      )
+      const projectPropertiesResponse = await fsxaApi.fetchProjectProperties({
+        locale
+      })
+      let projectProperties = null
+      if (
+        projectPropertiesResponse &&
+        Object.hasOwn(projectPropertiesResponse as Object, 'projectProperty')
+      ) {
+        projectProperties = (
+          projectPropertiesResponse as NormalizedProjectPropertyResponse
+        ).projectProperties
+      }
+      projectProperties = projectPropertiesResponse as ProjectProperties
+
+      return cachedProjectProperties.value[locale] || projectProperties
     } catch (error) {
       return null
```

Comment:
don't we need the error (with its message) here somehow?

Reply:
I'd suggest we make a follow-up issue of this. There has been a mistake with exporting the HttpError class of the content API. If we can fix that first, we can use it throughout the PWA-Template project for consistent error handling